src/cripts/Error.cc

```diff
@@ -29,7 +29,7 @@ Error::execute(Cript::Context *context)
 
 // These are static, to be used with the get() wrapper
 void
-Error::Message::_set(Cript::Context *context, Cript::string msg)
+Error::Message::_set(Cript::Context *context, const Cript::string &msg)
```

Comment:
Shouldn't this be a view?

Reply:
I don't remember now, there was probably some reason I made it a string ...